linera-views/src/scylla_db.rs

```diff
@@ -122,12 +123,13 @@ impl KeyValueStoreClient for ScyllaDbClientInternal {
         // There is probably a better way in ScyllaDB for downloading several keys than this one.
         let client = self.client.deref();
         let _guard = self.acquire().await;
-        let mut values = Vec::new();
+        let mut handles = Vec::new();
```

Comment:
No need to allocate a vector, I think:

```suggestion
        let handles = keys.into_iter().map(|key| Self::read_key_internal(client, key));
```

Reply:
Correct, thank you.